raiden_contracts/deploy/etherscan_verify.py

```diff
@@ -174,8 +175,17 @@ def etherscan_verify_contract(
             status = r['status']
```

Comment:
Not new in this PR, but a short comment what `content["status"] == "1"` means would be helpful.

Reply:
I added a comment.